### PR TITLE
Formatting organisations

### LINKED ISSUE #40: Format the organisations nicely down the page
At the moment the page is just showing the object values in `<p>` tags accross the page (including image links).

The organisations should be formatted nicer.

This would be a good point to get [bulma](http://bulma.io/) set up

### PR DESCRIPTION
**Pull request by shouston3**
Fixes #40 

Setting up sass and adding basic styling for each organisation using bulma
Setting up sass scripts
Moving `index.html` into build directory

### CODE REVIEW

File: src/public/js/timer.js [hunk 1]
```diff
@@ -4,20 +4,24 @@ export default (orgs) => {
...
-            <p>${org.name}</p>
-            <p>${org.image}</p>
-          `;
-        }).join('')
-      }
+      <div class="columns organisations">
+        ${
+          orgs.map((org) => {
+            return `
+              <div id="${org.id}" class="column organisation">
```
Comment (Jbarget) [inline src/public/js/timer.js]: having our organisations laid out on a page with 5em divs could take up a lot of real estate on the screen.

I was thinking that we have a [select field](http://bulma.io/documentation/elements/form/) with the organisations as the options within that. And the same for the repos and issues. 

That will keep UI and UX consistent throughout the 3 stages before showing the timer. and also allowing the user to change the organisation/repo/issues easily.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,9 @@
     "distmon": "nodemon -e html,js,json,css -w node_modules/app -w src -x 'npm run dist'",
     "start": "node src/api/start.js",
     "startmon": "nodemon -e html,js,json,css -w node_modules/app -w src -x 'npm start'",
-    "debug:server": "node --inspect src/api/app.js --debug-brk"
+    "debug:server": "node --inspect src/api/app.js --debug-brk",
+    "sass": "sass src/public/styles/base.scss:build/main.css",
```
Comment (Jbarget) [inline package.json]: we should run `npm run sass` in the dist and distmon script

File: src/public/js/timer.js (hunk 1 above)
Comment (Jbarget) [inline src/public/js/timer.js]: we can sort this out further down the line so dont worry about changing this

File: package.json (hunk 2 above)
Comment (shouston3) [inline package.json]: Changed, now the dist and distmon scripts should also build the sass

File: src/public/js/timer.js (hunk 1 above)
Comment (shouston3) [inline src/public/js/timer.js]: Cool I'm up for that, I just chose a height so that the organisations looked sensible on the page and `5ems` looked fine.

**Pull request merged**